loleaflet/src/control/Control.JSDialog.js

```diff
@@ -28,6 +28,10 @@ L.Control.JSDialog = L.Control.extend({
 
 	closeDialog: function(id) {
 		var builder = this.dialogs[id].builder;
+
+		if (this.dialogs[id].type === 'modalpopup')
```

Comment:
This condition is never met and I wonder if this is the right place or if there is additional stuff to be add in `closeBusyPopup()` or even in `hideBusy()`

Reply:
see line 118 (https://github.com/CollaboraOnline/online/pull/2624/files#diff-49fe2ec29af28fec557a60dbea747d3af4be9aa24d352ddbec4945e57098ba11L118)

in this.dialogs[id] there is no field type